core/server/models/stripe-customer-subscription.js

```diff
@@ -1,11 +1,48 @@
 const ghostBookshelf = require('./base');
 
+const CURRENCY_SYMBOLS = {
+    usd: '$',
+    aud: '$',
+    cad: '$',
+    gbp: '£',
+    eur: '€',
+    inr: '₹'
+};
+
 const StripeCustomerSubscription = ghostBookshelf.Model.extend({
     tableName: 'members_stripe_customers_subscriptions',
 
     customer() {
         return this.belongsTo('MemberStripeCustomer', 'customer_id', 'customer_id');
+    },
+
+    serialize(options) {
+        const defaultSerializedObject = ghostBookshelf.Model.prototype.serialize.call(this, options);
+
+        return {
+            id: defaultSerializedObject.subscription_id,
+            customer: {
+                id: defaultSerializedObject.customer_id,
+                // TODO? The customer is not fetched by default so these sometimes won't exist
```

Comment:
It used to be fetched by default in memebers-api, right? Don't think we'd be breaking much if we stop loading them by default, although it's technically a breaking change if we do. 

I'm also not sure we allow passing any includes for member's endpoints, so that's something that would have to be taken care if we stop fetching this relation.

Reply:
It did yeah!

My thinking was that in the members controllers we would just use `{withRelated: ['stripeSubscriptions', 'stripeSubscriptions.customer']}` in the config - that would keep the same functionality, but would allow for more performant queries when we _don't_ need the subscriptions - what do you think?

Reply:
Yup, the thinking is straight to the point! The the endpoints are not stable yet, and whole members is beta so I think even if we start returning members without subscriptions by default we should be fine :)

Reply:
To be extra clear, would be cool to keep the functionality as is - with subscriptions by default, but we could discuss with product changing this to not return them by default which I'd be in favor of :)

Reply:
Oh yeah, I totally think the HTTP API should be kept as it is for now! If you look at the `members.js` controller file, you'll see I've updated all the methods to use `withRelated` :) 

> but we could discuss with product changing this to not return them by default which I'd be in favor of :)

Yeah I _think_ I would to